client/src/index.js

```diff
@@ -1,22 +1,35 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import styles from './Style.module.css';
 
 import ChatLink from './pages/chatlink';
 import Messaging from './pages/messaging';
 import PoweredBy from './components/PoweredBy';
+import { ThemeProvider, ThemeContext } from './ThemeContext.js';
 
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
+const App = () => {
+  const [darkMode, setdarkMode] = useContext(ThemeContext);
+
+  return (
+    <body className={!darkMode && styles.lightMode}>
```

Comment:
This will become invalid html syntax. There is already a `body` tag outside.
Not sure why do you need a `body` tag here

https://github.com/muke1908/chat-e2ee/blob/a0c5a3b98574eb52894cd78a066db066614d9002/client/public/index.html#L26

Reply:
If I remove the body tag this is what I get
![Screenshot from 2020-08-12 09-14-07](https://user-images.githubusercontent.com/62807661/89973262-8c3c3480-dc7d-11ea-9fee-8942b1817bc4.png)

Reply:
I replaced it with the body tag with a div tag